cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -295,6 +311,37 @@ mkTracers traceOptions tracer = Tracers
         $ withName "LocalTxSubmissionProtocol" tracer
       }
 
+-- | get information about a chain fragment
+-- returns a triple of
+--     * number of slots
+--     * number of blocks
+--     * ratio of copied blocks over announced blocks
+chainInformation :: forall block . AF.HasHeader block
+                 => AF.AnchoredFragment block -> (Integer, Integer, Rational)
```

Comment:
Is `Integer` (arbitrary-sized, signed) the best type for this? Why not `Word64` or `Word`?

Reply:
not the best choice, indeed.
and, could be a record as the two first items can be swaped, which would have some impact!

Reply:
I would also advocate for a this being a record :slightly_smiling_face:

Reply:
it's now a record, and using Word64.